src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -218,7 +223,31 @@ private void addQualificationPlanToProject(ModelProject project, Model.Qualifica
          project.GetOrCreateClassifiableFor<TClassifiableWrapper, TSubject>(subject);
       }
 
-      private Task<Model.Simulation[]> allSmulationsFrom(Simulation[] snapshotSimulations, ModelProject project) => _simulationMapper.MapToModels(snapshotSimulations, project);
+      private async Task<IEnumerable<Model.Simulation>> allSmulationsFrom(Simulation[] snapshots, ModelProject project)
```

Comment:
Also not running simulation in //. This makes life easier when looking at the log file